### PR TITLE
feat(i18n): adding i18n for signin page

### PR DESCRIPTION
**Pull request by GabrielNSilva**
I'll work on the i18n of the pages. Should I sending the PRs on demand in order to update the project for non-English speaking users, or finish the whole work to open the PR?

### CODE REVIEW

File: app/public/locales/pt-BR/login.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  "generatingQrCode": "Gerando QrCode...",
+  "scanQrCode": "Escaneie este QrCode ou copie + cole em tua carteira. Ou clique para logar com tua carteira do navegador.",
+  "clickToConnect": "Clique para conectar",
+  "compatibleWallets": "Carteiras compatíveis",
+  "recommendedWallet": "Carteira recomendada",
+  "otherWallets": "Outras carteiras",
+  "install": "Instalar",
+  "open": "Abrir",
+  "use": "Use",
+  "instead": ""
```
Comment (rolznz) [inline app/public/locales/pt-BR/login.json:13]: Instead is an empty string, is this intentional?

File: app/public/locales/en/home.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "orangePill": "Orange pill the world around you.",
```
Comment (rolznz) [inline app/public/locales/en/home.json:2]: It looks like these translations aren't used yet in this PR, are you able to consume them in the index file?

**Comment by rolznz** [comment]
@GabrielNSilva awesome work! it would be great to open one PR per section of the app going forwards in case we have other team members working on translations as well. I will hopefully be adding some more support for translations soon, probably continuing on the recipient journey side first, so please do let us know. I have started a discord channel here, if you would like to join: https://discord.gg/3QkV7ny2EW

File: app/public/locales/pt-BR/login.json (hunk 1 above)
Comment (GabrielNSilva) [reply app/public/locales/pt-BR/login.json:13]: Yes, in portuguese we dont use any expression in the end of that phrase

File: app/public/locales/en/home.json (hunk 2 above)
Comment (GabrielNSilva) [reply app/public/locales/en/home.json:2]: I'll remove this file in this PR, and add it in a future.

**Pull request merged**